src/Nethermind/Nethermind.AuRa.Test/AuRaBlockProducerTests.cs

```diff
@@ -180,7 +180,7 @@ private async Task<TestResult> StartStop(bool processingQueueEmpty = true, bool
                 _blockTree.NewBestSuggestedBlock += Raise.EventWith(new BlockEventArgs(Build.A.Block.TestObject));
             }
             
-            await Task.Delay(_stepDelay * 10);
+            await Task.Delay(_stepDelay * 20);
```

Comment:
instead of delay please use conditional waiting to stop the teat as soon as it succeeds